File: CS_Collections_benchmark/BenchmarkResults.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CS_Collections_benchmark
{
    class BenchmarkResults
    {
        public double AddTime { get; set; }
        public double FindTime { get; set; }
        public double RemoveTime { get; set; }

        public override string ToString()
        {
            return string.Format("Add: {0}\n" +
                "Find: {1}\n" +
                "Remove: {2}\n",
                AddTime, FindTime, RemoveTime);
        }

    }
}

File: Java_Collections_Benchmark/src/com/company/BenchmarkResults.java
package com.company;

public class BenchmarkResults {

    public double CreateAndAddTime;
    public double AddTime;
    public double FindTime;
    public double RemoveTime;

    @Override
    public String toString() {
        return String.format("Create and add: %1$s\n" +
                "Add: %2$s\n" +
                "Find: %3$s\n" +
                "Remove: %4$s\n",
                CreateAndAddTime, AddTime, FindTime, RemoveTime);
    }
}
